### ISSUE TITLE
Add show+raise+openChatTab click handler on chat icon

### ISSUE THREAD

**Issue by simonbcn** [foudfou/FireTray#60]
I have updated to version 0.4.3.
It displays an icon green for the chat. I suposse that this color means that I'm connected. But when I click on it, it does nothing. Shouldn't it open the chat window?

**Comment by RalfJung** [comment]
I agree that "show on click" is such a standard behavior for tray icons (FireTray itself implements it for its "main" icon) that one really expects the chat icon to do the same: Clicking on it, show show Thunderbird. If it can show the chat tab, even better. But currently, when the chat icon blinks, I have to click *another icon* to read the chat message. That's kind of confusing.
